source/nodejs/adaptivecards-designer/src/containers/cortana/cortana-container.ts

```diff
@@ -1,11 +1,18 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Just a general question: Do we want to reset the `containerThemeChoicePicker` whenever someone changes container?

Reply:
I don't think so.